context/app/static/js/components/Detail/VisualizationWrapper/VisualizationWrapper.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Suspense } from 'react';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import { StyledSectionContainer, StyledHeader, StyledHeaderText } from '../Visualization/style';
+import { VisualizationBackground } from './style';
+
+const Visualization = React.lazy(() => import('../Visualization'));
+
+function VisualizationWrapper(props) {
+  const { vitData } = props;
+
+  return (
+    <Suspense
+      fallback={
+        <StyledSectionContainer id="visualization">
+          <StyledHeader>
+            <StyledHeaderText variant="h4" component="h2">
+              Visualization
+            </StyledHeaderText>
+          </StyledHeader>
+          <VisualizationBackground>
+            <CircularProgress />
+          </VisualizationBackground>
```

Comment:
Good, so the please-wait will use as much height as the final result, right?

Reply:
Yes.